docs/deprecations.md

```diff
@@ -133,3 +133,9 @@ Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2
 ## `awsKmsKeyArn` references
 
 Please use `provider.kmsKeyArn` and `functions[].kmsKeyArn`. `service.awsKmsKeyArn` and `functions[].awsKmsKeyArn` will be removed with v3.0.0
+
+<a name="PARAMETERIZED_ARGUMENT"><div>&nbsp;</div></a>
```

Comment:
Deprecation notice looks good, however, I have a question related to the placement of new deprecations in that doc. I've always defaulted to putting new deprecations to the top of the list, because it's the first thing you see when opening the deprecation list, but I can also see an argument on placing it at the bottom. What is your opinion on that? I'm accepting as I don't have a strong opinion here but I figured it's worth asking

Reply:
Good point @pgrzesik 👍  I didn't notice that. Ideally most recent deprecations are at the top of the list

Reply:
Sure I can do that, I wasn't aware of the convention - I just added one at the bottom.